inscriptions/ISic002998.xml

```diff
@@ -195,24 +200,31 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI332328">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Λυσίμα-<lb n="2"/>χος, Με-<lb n="3"/>λίσσας πα-<lb n="4"/>ρὰ Παίδε-<lb n="5"/>σσι καὶ ἅμ
-                    <supplied reason="lost">α(?)</supplied>
-                    <lb n="6"/>Ἄνναι ἱερεί-<lb n="7"/>ας, γεγέννηται<lb n="8"/>μετὰ <supplied reason="lost">
-                    Εὐτρ(?)</supplied>ό-<lb n="9"/>που κα<supplied reason="lost">ὶ</supplied> Γο-<lb n="10"/>νατίας καὶ
-                    <lb n="11"/>τῶν <supplied reason="lost">ἐκ</supplied>είνου<lb n="12"/>συμβιωτῶν<lb n="13"/>καὶ
-                    Ἔρωτος<lb n="14"/>ναβλᾶ. vacat
-                    <lb n="15"/>
-                    <surplus>²pentalpha</surplus>²
-                    <lb n="16"/>
+                    <lb n="1"/>Λυσίμα
+                    <lb n="2" break="no"/>χος Με
+                    <lb n="3" break="no"/>λίσσας πα
+                    <lb n="4" break="no"/>ρὰ Παίδε
+                    <lb n="5" break="no"/>σσι καὶ τῇ
+                    <lb n="6"/>Ἄννᾳ ἱερε
+                    <lb n="7" break="no"/>ίας γεγέ<unclear>ν</unclear><reg>ν</reg>ηται
```

Comment:
again, I think this needs to be:`<choice><reg>γεγέ<unclear>ν</unclear>νηται</reg><orig>γεγέ<unclear>ν</unclear>ηται</orig><choice>` ?